socorro/collector/throttler.py

```diff
@@ -32,20 +33,11 @@ class LegacyThrottler(RequiredConfig):
               and d.get("ProcessType", "browser") == "browser"''', None),
         # 100% of crashes with comments
         ("Comments", '''lambda x: x''', 100),
-        # 100% of all aurora, beta, esr channels
-        ("ReleaseChannel", '''lambda x: x in ("aurora", "beta", "esr")''', 100),
-        # 100% of all crashes that report as being nightly
-        ("ReleaseChannel", '''lambda x: x.startswith('nightly')''', 100),
-        # 10% of Firefox
-        ("ProductName", 'Firefox', 10),
-        # 100% of Fennec
-        ("ProductName", 'Fennec', 100),
-        # 100% of all alpha, beta or special
-        ("Version", r'''re.compile(r'\..*?[a-zA-Z]+')''', 100),
-        # 100% of Thunderbird & SeaMonkey
-        ("ProductName", '''lambda x: x[0] in "TSC"''', 100),
-        # reject everything else
-        (None, True, 0)
+        # 10% of Firefox release
+        ("*", '''lambda d: d.get("ProductName") == "Firefox" and
+            d.get("ReleaseChannel") == "release"''', 10),
+        # accept everything else
+        (None, True, 100)
```

Comment:
while I admit that the protection was weak before, this really open Socorro wide for a DOS attack. With this change the collectors will accept any junk submitted and send it on to the processors.